sql/core/src/main/scala/org/apache/spark/sql/DataStreamReader.scala

```diff
@@ -100,6 +100,7 @@ class DataStreamReader private[sql](sqlContext: SQLContext) extends Logging {
     val resolved = ResolvedDataSource.createSource(
       sqlContext,
       userSpecifiedSchema = userSpecifiedSchema,
+      partitionColumns = Array.empty[String],
```

Comment:
Wait second... I think we only ever give the partition columns to a DataSource when it is coming from the hive metastore.  Given that, I don't know if we actually need to do this at all for streaming.  Could we simplify this?